manifests/base/crds/argoproj.io_eventbuses.yaml

```diff
@@ -2,13 +2,13 @@
 apiVersion: apiextensions.k8s.io/v1
```

Comment:
Please revert it, we don't use `eventbuses` but `eventbus`. We have `plural exceptions` at https://github.com/argoproj/argo-events/blob/master/hack/update-codegen.sh#L41, but somehow your local code-generator version screwed it.